samcli/local/docker/lambda_debug_settings.py

```diff
@@ -108,7 +108,8 @@ def get_debug_settings(debug_port, debug_args_list, _container_env_vars, runtime
                 + ["--no-lazy", "--expose-gc"]
                 + ["/var/runtime/index.js"],
                 container_env_vars={
-                    "NODE_PATH": "/opt/nodejs/node_modules:/opt/nodejs/node10/node_modules:/var/runtime/node_modules",
+                    "NODE_PATH": "/opt/nodejs/node_modules:/opt/nodejs/node10/node_modules:/var/runtime/node_modules:"
+                    "/var/runtime:/var/task",
```

Comment:
Just curious, is this a pylint rule that requires splitting strings on separate lines?

Reply:
yes